src/game_objects/grid.cpp

```diff
@@ -53,15 +53,15 @@ void Grid::setup_listeners() {
 }
 
 void Grid::update() {
-    if (grid[hoverX][hoverZ] != hover) {
+    if (grid[hoverZ][hoverX] != hover) {
```

Comment:
Due to the nature of the way I import the map data, the grid uses [row][column], so it actually works more like [z][x]. Sorry for any confusion. I'll need to figure out a cleaner way to make sure ppl don't get confused